docs/reference/errors-and-warnings/NU5128.md

```diff
@@ -0,0 +1,80 @@
+---
+title: NuGet Warning NU5125
+description: NU5125 Warning code
+author: zivkan
+ms.date: 09/06/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5128
+---
+
+# NuGet Warning NU5128
+
+## Scenario 1
+
+<pre>Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder do not have exact matches in the other location. Consult the list of actions below:
+- Add a dependency group for .NETStandard2.0 to the nuspec</pre>
+
+### Issue
+
+The `lib/<tfm>/` or `ref/<tfm>/` directory in the package contains at least one file for the Target Framework Moniker (TFM) specified in the warning message. However, no dependency group exists for this TFM in the `nuspec` file. This may cause package consumers who use dependency information to believe the package is not compatible with the TFM.
+
+### Solution
+
+* Let NuGet automate `lib/` files and dependency groups
```

Comment:
these 2 headers are great.  perhaps better if we call them:
1) call pack on a project (and nuget automatically creates nuspec)
2) doing hand creation of a nuspec 

(better terminology can be used, likely)